include/libp2p/protocol/gossip/gossip.hpp

```diff
@@ -48,13 +49,25 @@ namespace libp2p::protocol::gossip {
     bool echo_forward_mode = false;
 
     /// Read or write timeout per whole network operation
-    unsigned rw_timeout_msec = 10000;
+    std::chrono::milliseconds rw_timeout_msec{10000};
 
-    unsigned message_cache_lifetime_msec = 120000;
+    /// Lifetime of a message in message cache
+    std::chrono::milliseconds message_cache_lifetime_msec{120000};
 
-    unsigned seen_cache_lifetime_msec = 60000;
+    /// Topic's message seen cache lifetime
+    std::chrono::milliseconds seen_cache_lifetime_msec{60000};
 
-    unsigned heartbeat_interval_msec = 1000;
+    /// Heartbeat interval
+    std::chrono::milliseconds heartbeat_interval_msec{1000};
+
+    /// Ban interval between dial attempts to peer
+    std::chrono::milliseconds ban_interval_msec{60000};
```

Comment:
The same idea as for the value of 3600000 ms
```suggestion
    std::chrono::milliseconds ban_interval_msec{std::chrono::minutes(1)};
```